diff --color -uNr proot/src/arch.h proot.mod/src/arch.h
--- proot/src/arch.h	2021-12-24 16:12:03.555700162 +0800
+++ proot.mod/src/arch.h	2021-12-24 16:25:53.047699845 +0800
@@ -101,7 +101,7 @@
     #define OFFSETOF_STAT_GID_32 0
     #define EM_ARM 40
 
-    #define LOADER_ADDRESS 0x10000000
+    #define LOADER_ADDRESS 0x20000000
 
     #define EXEC_PIC_ADDRESS   0x0f000000
     #define INTERP_PIC_ADDRESS 0x1f000000
